qa/SidechainTestFramework/account/smart_contract_resources/contracts/folderthing/Solfile.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract ExampleERC20Contract is ERC20 {
```

Comment:
why do we have this contract declared twice here and in `contracts/folderthing/Othercontract.sol`?

Reply:
This was for testing purposes - the way smart contracts are searched for, every contract needs to have a unique name. I did this to check if / how this works and if the error returned makes sense.